pkgs/by-name/pa/passepartui/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  fetchFromGitHub,
+  fetchpatch,
+  nix-update-script,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "passepartui";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "kardwen";
+    repo = "passepartui";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-3visFhsyXwKTklCpGMwtxeYJyXyCKRc/PwTXD2QzFog=";
+  };
+
+  cargoPatches = [
+    # https://github.com/kardwen/passepartui/pull/3
+    (fetchpatch {
+      url = "https://github.com/kardwen/passepartui/commit/a77e01facc27c5019c1511b41cc07a7231eb84c1.patch";
+      hash = "sha256-BWI9KkHELgJmWcdSaS51yj+W+11PdBpncwyf9SwYhHw=";
+    })
+  ];
+
+  cargoHash = "sha256-6CTbJ0o9/HzCp9DK1YnPO3mZ1CINgja/plwDRShn+Ls=";
+
+  passthru = {
+    updateScript = nix-update-script { };
```

```suggestion
    updateScript = nix-update-script {
      extraArgs = [
        "--version-regex"
        "'v(\d+\.\d+\.\d+)'"
      ];
    };
```

It might be worth adding a version regex here? Not sure, i am a a bit inexperienced with update scripts, but might be worth considering.

---

This basically the default regex for `nix-update`. It should be able to figure out new versions without this